### DISCUSSION TITLE
How do I load a pre-trained model in c++?

### DISCUSSION THREAD

**Discussion by Rian-Jo** [google/brax#444] [Q&A] [answered]
Thanks for your great work.

I want to deploy the pretrained policy to the c++ project. How can i load the policy trained in python by brax in c++ project? Are mjx or brax supports c++ libraries to implement the policy model?

Thank you.

**Comment by btaba** [comment, marked as answer]
Hi @Rian-Jo , there are a few options out there, but none that seem super great. Here's one example https://github.com/google/jax/tree/main/jax/experimental/jax2tf

**Reply by Rian-Jo** [reply]
thank you for the reply.  I hope the project implement the c++ deployment someday :)
